src/styles/scss/index.scss

```diff
@@ -1,4 +1,8 @@
-@import 'base/variables';
-@import 'base/elements';
+@import "base/variables";
```

Comment:
How do you gals/guys approach imports?
I always make a case to import things such `variables` on every file, where needed instead the "main" file.
What do you think?